DepositUI/Startup.cs

```diff
@@ -26,6 +22,8 @@ public Startup(IConfiguration configuration)
         // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
         public void ConfigureServices(IServiceCollection services)
         {
+            services.AddScoped<IDepositService, DepositService>();
+            services.AddAutoMapper(typeof(MapperProfile));
             services.AddRazorPages();
             services.AddServerSideBlazor();
```

Comment:
We should use the client-side project. Consider ability to change the project, because rendering of pages works differently. For more details see here: https://habr.com/ru/post/546414/ https://docs.microsoft.com/ru-ru/aspnet/core/blazor/hosting-models?view=aspnetcore-5.0 https://blazor-university.com/overview/blazor-hosting-models/